packages/compass-crud/src/components/bulk-update-dialog.tsx

```diff
@@ -163,6 +163,7 @@ export default function BulkUpdateDialog({
   const darkMode = useDarkMode();
 
   const [text, setText] = useState(updateText);
+  const [wasOpen, setWasOpen] = useState(isOpen);
```

Comment:
Should this be a `useRef` as it isn't involved in the rendering life cycle?
If someone calls `setWasOpen` I think we don't want to re-render.

Reply:
I switched to the usePrevious() pattern we already use elsewhere.